packages/integration-tests/projects/suite-web/tests/metadata/account-metadata.test.ts

```diff
@@ -5,8 +5,8 @@ import { rerouteMetadataToMockProvider, stubOpen } from '../../stubs/metadata';
 
 // fixture contains number of request that given provider needs go through this test scenario
 const fixtures = [
-    { provider: 'dropbox', numberOfRequests: 24 },
-    { provider: 'google', numberOfRequests: 10},
+    { provider: 'dropbox', numberOfRequests: [24, 25] },
+    { provider: 'google', numberOfRequests: [10, 12] },
```

Comment:
What are the indexes in the `numberOfRequests` array for? A comment describing it would be helpful.

Reply:
or change it to an object so it self descriptive (if possible)

Reply:
these are number of requests to be asserted at a certain point of time during the test. If I was able to come up with a proper name for these two points I would have certainly done it that way.

Reply:
and there is a comment on line 6

Reply:
The comment between L96 to L99 gave me a bit more context. As @tomexx said, an object would be a bit more descriptive than an array. But that's just a nitpick.